include/Mac.h

```diff
@@ -176,6 +200,10 @@ class Mac : public GenericHashEntry {
     else
       return(false);
   }
+  inline void incTalkersAsClient()  { talkers.as_client++; }
```

Comment:
word Talkers is not self-explanatory per se. Use a name which explains the behavior of these functions.